src/django/api/tests.py

```diff
@@ -7845,7 +7877,10 @@ def setUp(self):
         super(ParentCompanyTestCase, self).setUp()
         self.url = reverse('facility-list')
 
-    def test_submit_parent_company_no_match(self):
+    @patch('api.geocoding.requests.get')
```

Comment:
Updated a bunch of the tests to mock the external geocoding API (preventing external dependency in tests)